tools/make/docs.mk

```diff
@@ -1,14 +1,36 @@
-DOCS_DIR := docs
-DOCS_OUTPUT_DIR := docs/html
+DOCS_DIR := docs/latest
```

Comment:
curious why this is special, and we cannot use `latest` as a `RELEASE_VERSION` ?

Reply:
Thanks for pointing this out, I have updated.